CONTRIBUTING.md

```diff
@@ -69,19 +69,9 @@ The output of the tests will be who under the [DEBUG CONSOLE](https://code.visua
 ### Test from Github Codespaces
 
 To run unit tests from within a Codespace:
-
-```
-## Install missing libraries and framebuffer x-server.
-## (Runs automatically in Github Codepsaces because
-## of `.devcontainer/devcontainer.json`).
-sudo apt-get install libxshmfence1 xvfb -y
-## Launch framebuffer x-server and force tests to use
-## framebuffer x-server.
-sudo Xvfb -ac :99 -screen 0 1280x1024x16 &
-export DISPLAY=:99
-## Run tests.
-npm test
-```
+1. Switch to the [Debug and Run](https://code.visualstudio.com/docs/editor/debugging) panel.
```

Comment:
I see no harm in keeping the CLI instructions next to the new instructions.

Reply:
Added it back and updated it to be the same as what is run in the CI, but leaved the one of Debug and Run panel as the prefered one because it creates a new window where we can see the tests running and debug them if required